front/config.form.php

```diff
@@ -54,6 +54,13 @@
       }
    }
    Session::addMessageAfterRedirect(__('The cache has been emptied.', 'barcode'));
+} else if (isset($_POST['dropLogo'])) {
+   if (is_dir(GLPI_PLUGIN_DOC_DIR.'/barcode')) {
+      if (is_file(GLPI_PLUGIN_DOC_DIR.'/barcode/logo.png')) {
+         unlink(GLPI_PLUGIN_DOC_DIR.'/barcode/logo.png');
+      }
+   }
```

Comment:
First check is useless.
```suggestion
   if (is_file(GLPI_PLUGIN_DOC_DIR.'/barcode/logo.png')) {
      unlink(GLPI_PLUGIN_DOC_DIR.'/barcode/logo.png');
   }
```